Makefile

```diff
@@ -182,11 +189,10 @@ verify-mockgen: mockgen diff
 verify-manifests: manifests diff
 verify: verify-codegen verify-mockgen verify-manifests
 
-
 # before running release, bump the version in OLM_VERSION and push to master,
 # then tag those builds in quay with the version in OLM_VERSION
 release: ver=$(shell cat OLM_VERSION)
-release:
+release: manifests
```

Comment:
this seems like a major regression if the manifest generation went missing. thanks for catching this.